docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Docker build file to create a centos image for building rpm packages
+
+FROM centos:7
+MAINTAINER andreas-bok-sociomantic <andreas-bok@sociomantic.com>
+
+RUN yum install -y rpm-build zlib-devel libtool openssl \
+    openssl-devel automake autoconf gcc-c++ && \
+    yum clean all && \
+    rm -r -f /var/cache/*
```

Comment:
Ah. See your point since the container is transient... :)

`yum clean all && \` can be omitted as well